src/redux/boards/reducer.js

```diff
@@ -35,6 +35,22 @@ export default function reducer(state = Immutable(defaultState), action) {
     case actions.GET_BOARD_FAILURE: {
       return state.merge({ loading: false, error: action.payload.error });
     }
+    case actions.DELETE_BOARD: {
+      return state.merge({ loading: true });
+    }
+    case actions.DELETE_BOARD_SUCCESS: {
+      const deletedBoardIndex = state.list.findIndex(board => board.id === action.payload.boardId);
+      const newList = state.list
+        .slice(0, deletedBoardIndex)
+        .concat(state.list.slice(deletedBoardIndex + 1, state.list.length));
```

Comment:
Mmm IMHO this will get more documented if this list operation belongs to a function

Reply:
move this to array utils

Reply:
I'd also write this code using the spread operator `...` instead of using `.concat()`